src/test/kotlin/com/zepben/evolve/services/network/testdata/FillFields.kt

```diff
@@ -247,7 +247,7 @@ fun Structure.fillFields(service: NetworkService, includeRuntime: Boolean = true
 fun Location.fillFields(service: NetworkService, includeRuntime: Boolean = true): Location {
     (this as IdentifiedObject).fillFieldsCommon(service, includeRuntime)
 
-    mainAddress = StreetAddress("1234", null)
+    mainAddress = StreetAddress("1234", null, "5678", null)
```

Comment:
`fillFields` should be filling all values, not specifying `null`. Can you check to make sure the `townDetail` can be updated as it should have never been null in the first place.

Reply:
If I specify default values for all fields of `StreetAddress`, is this better:
```
mainAddress = StreetAddress()
```

Reply:
New revision uses above code.